etc/systemd/system/zfs-import-cache.service.in

```diff
@@ -4,6 +4,7 @@ DefaultDependencies=no
 Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=cryptsetup.target
+Before=dracut-mount.service
```

Comment:
Won't this cause issues with the service for people who have systemd, but not dracut?